js/repeater-list.js

```diff
@@ -538,7 +538,7 @@
 			$col.html(checkBoxMarkup);
 		}
 
-		if (this.viewOptions.list_columnRendered) {
+		if (this.viewOptions.list_columnRendered && !(columns[columnIndex].property === '@_CHECKBOX_@' || columns[columnIndex].property === '@_ACTIONS_@')) {
```

Comment:
switch the order. if the column is checkbox or action it doesn't matter if there is a custom render.